src/bench/merkle_root.cpp

```diff
@@ -11,16 +11,15 @@
 
 static void MerkleRoot(benchmark::Bench& bench)
 {
-    FastRandomContext rng(true);
-    std::vector<uint256> leaves;
-    leaves.resize(9001);
+    FastRandomContext rng{/*fDeterministic=*/true};
+    std::vector<uint256> leaves(9000); // even number of leaves
```

Comment:
Wouldn't it be better to benchmark the less-performant input?

Reply:
After this change all call-sites are pre-sized to even number of leaves, so this reflects the usage better.
I could also round to `9002` to mimic the new `leaves.reserve((9001 + 1) & ~1ULL)` behavior, if you thinks that's more representative.